src/ui/UIOMultilingualDemo.js

```diff
@@ -0,0 +1,74 @@
+"use strict";
+
+(function ($, fluid) {
+
+    fluid.defaults("fluid.uiOptions.prefsEditor.multilingualDemo", {
+        gradeNames: ["fluid.uiOptions.prefsEditor"],
+        terms: {
+            "messagePrefix": "node_modules/infusion/src/framework/preferences/messages",
+            "templatePrefix": "node_modules/infusion/src/framework/preferences/html"
+        },
+        "tocTemplate": "node_modules/infusion/src/components/tableOfContents/html/TableOfContents.html",
+        "tocMessage": "node_modules/infusion/src/framework/preferences/messages/tableOfContents-enactor.json",
+        "ignoreForToC": {
+            "overviewPanel": ".flc-overviewPanel"
+        },
+        model: {
+            locale: "en",
+            direction: "ltr"
+        },
+        listeners: {
+            "onPrefsEditorReady.addLanguageAttributesToBody": {
+                func: "fluid.uiOptions.prefsEditor.multilingualDemo.addLanguageAttributesToBody",
+                args: ["{that}.prefsEditorLoader.prefsEditor.container", "{that}.model.locale", "{that}.model.direction"]
+            }
+        },
+        distributeOptions: {
+            "messageLoaderLocale": {
+                target: "{that messageLoader}.options.model",
+                record: {
+                    resourceLoader: {
+                        locale: "{multilingualDemo}.model.locale"
+                    }
+                }
+            },
+            "relayOnCreateToc": {
```

Comment:
"onCreateTOC" should be fired when "{messageLoader}.events.onResourcesLoaded" is fired.

But this doesn't solve the problem that the toc header is showing "[Object Object]" when the initial language is Spanish.

Reply:
This review comment can be ignored because "onCreateTOCReady", the event that controls the creation of TOC, is [an aggregate event](https://github.com/fluid-project/infusion/blob/master/src/framework/preferences/js/Enactors.js#L482-L487) including "onMessagesLoaded" so the timing of when "onCreateTOC" is fired becomes less sensitive.